packages/flutter/lib/src/material/switch.dart

```diff
@@ -325,6 +337,7 @@ class _SwitchRenderObjectWidget extends LeafRenderObjectWidget {
       ..onChanged = onChanged
       ..textDirection = Directionality.of(context)
       ..additionalConstraints = additionalConstraints
+      .._drag.dragStartBehavior = dragStartBehavior
```

Comment:
since you have a dragStartBehaviour get/set pair in _RenderSwitch, you can just use that here, that'd be cleaner